db/migrate/20220722215911_change_text_profile_column_name.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeTextProfileColumnName < ActiveRecord::Migration[7.0]
+  def change
+    add_column :partner_profiles, :client_capacity, :string
+    add_column :partner_profiles, :primary_contact_name, :string
+    add_column :partner_profiles, :primary_contact_phone, :string
+    add_column :partner_profiles, :primary_contact_mobile, :string
+    add_column :partner_profiles,  :primary_contact_email, :string
+
+    ::Partners::Partner.all.find_each do |partner_profile|
+      partner_profile.client_capacity = partner_profile.max_serve
+      partner_profile.program_contact_name = partner_profile.primary_contact_name
+      partner_profile.program_contact_phone = partner_profile.primary_contact_phone
+      partner_profile.program_contact_mobile = partner_profile.primary_contact_mobile
+      partner_profile.program_contact_email = partner_profile.primary_contact_email
+    end 
+  end
```

Comment:
I decided create new columns and copy old values to them, because it is safely.
After deploy this, we can create a new PR to remove them.